core/module_specifier.rs

```diff
@@ -80,3 +108,85 @@ impl PartialEq<String> for ModuleSpecifier {
     &self.to_string() == other
   }
 }
+
+#[cfg(test)]
+mod tests {
+  use super::*;
+
+  #[test]
+  fn test_absolute() {
+    if cfg!(target_os = "windows") {
+      assert_eq!(
+        ModuleSpecifier::resolve_from_cwd(r"C:/deno/tests/006_url_imports.ts")
+          .unwrap()
+          .to_string(),
+        "file:///C:/deno/tests/006_url_imports.ts",
+      );
+      assert_eq!(
+        ModuleSpecifier::resolve_from_cwd(r"C:\deno\tests\006_url_imports.ts")
+          .unwrap()
+          .to_string(),
+        "file:///C:/deno/tests/006_url_imports.ts",
+      );
+    } else {
+      assert_eq!(
+        ModuleSpecifier::resolve_from_cwd("/deno/tests/006_url_imports.ts")
+          .unwrap()
+          .to_string(),
+        "file:///deno/tests/006_url_imports.ts",
+      );
+    }
+  }
+
+  #[test]
+  fn test_relative() {
+    // Assuming cwd is the deno repo root.
+    let cwd = std::env::current_dir().unwrap();
+    let cwd_string = String::from(cwd.to_str().unwrap()) + "/";
+    let expected_url = format!(
+      "file://{}{}",
+      cwd_string.as_str(),
+      "tests/006_url_imports.ts"
+    ).replace("\\", "/");
+
+    if cfg!(target_os = "windows") {
+      assert_eq!(
+        ModuleSpecifier::resolve_from_cwd(r"/tests/006_url_imports.ts")
```

Comment:
Should this start with ./ instead of just / ?

Reply:
TBH I'm not sure... according to [comment you linked](https://github.com/denoland/deno/pull/987#issuecomment-438573356) it is a relative path on windows.

Reply:
Okay, tests suggests that it is actually absolute path:
```
---- module_specifier::tests::test_relative stdout ----
thread 'module_specifier::tests::test_relative' panicked at 'assertion failed: `(left == right)`
  left: `"file:///C:/tests/006_url_imports.ts"`,
 right: `"file://C:/deno/tests/006_url_imports.ts"`', ../../core\module_specifier.rs:153:7
```

Reply:
"/foo/bar" is relative to the current drive, is what @piscisaureus meant.

Reply:
Alright, thanks for explanation. I just installed Parallels, so I'll double check everything on my own